opensrp-chw/src/main/java/org/smartregister/chw/activity/ChildProfileActivity.java

```diff
@@ -161,10 +161,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         super.onCreateOptionsMenu(menu);
-        menu.findItem(R.id.action_sick_child_form).setVisible(ChwApplication.getApplicationFlavor().hasChildSickForm() && flavor.isChildOverTwoMonths(((CoreChildProfilePresenter) presenter).getChildClient()));
+        menu.findItem(R.id.action_sick_child_form).setVisible(ChwApplication.getApplicationFlavor().hasChildSickForm()
+                && flavor.isChildOverTwoMonths(((CoreChildProfilePresenter) presenter).getChildClient())
+                && !ChwApplication.getApplicationFlavor().useThinkMd());
```

Comment:
why not use `flavor` on the line above

Reply:
flavor object is used to provide different implementation for methods, here we only needs to check the thinkmd support, we can check that from ChwApplication.getApplicationFlavor().useThinkMd(). so no need the flavor to use.